benchmark/single-source/DataBenchmarks.swift

```diff
@@ -414,3 +547,104 @@ public func run_AppendDataLargeToLarge(_ N: Int) {
     let other = sampleData(.large)
     benchmark_AppendData(N, data, other)
 }
+
+@inline(never)
+public func run_createEmpty(_ N: Int) {
+    for _ in 0..<100000 * N {
+        blackHole(Data())
+    }
+}
+
+@inline(never)
+public func run_createSmall(_ N: Int) {
+    for _ in 0..<100000 * N {
+        blackHole(sampleData(.small))
+    }
+}
+
+@inline(never)
+public func run_createMedium(_ N: Int) {
+    for _ in 0..<10000 * N {
+        blackHole(sampleData(.medium))
+    }
+}
+
+@inline(never)
+public func run_createLarge(_ N: Int) {
+    blackHole(sampleData(.veryLarge))
+}
+
+@inline(never)
+public func run_createEmptyArray(_ N: Int) {
+    for _ in 0..<100000 * N {
+        blackHole(Data([]))
+    }
+}
+
+@inline(never)
+public func run_createSmallArray(_ N: Int) {
+    for _ in 0..<100000 * N {
+        blackHole(Data([0, 1, 2, 3, 4, 5, 6]))
+    }
+}
+
+@inline(never)
+public func run_createMediumArray(_ N: Int) {
+    for _ in 0..<10000 * N {
+        blackHole(Data([0, 1, 2, 3, 4, 5, 6, 0, 1, 2, 3, 4, 5, 6, 0, 1, 2, 3, 4, 5, 6, 0, 1, 2, 3, 4, 5, 6]))
+    }
+}
+
```

Comment:
@milseman What other converters do we want to test here? Should we test the Foundation overlay variations as well?

Reply:
Up to you. Next release, we'll want to unify them as part of some upcoming string spring cleaning.